web/src/main/java/org/fao/geonet/Geonetwork.java

```diff
@@ -560,14 +560,15 @@ private boolean checkDBWrite() {
      */
     private void createSiteLogo(String nodeUuid, ServiceContext context, Path appPath) {
         try {
-            Path logosDir = Resources.locateLogosDir(context);
-            Path logo = logosDir.resolve(nodeUuid + ".png");
-            if (!Files.exists(logo)) {
+            final Resources resources = context.getBean(Resources.class);
+            Path logosDir = resources.locateLogosDir(context);
+//            Path logo = logosDir.resolve(nodeUuid + ".png");
+//            if (!Files.exists(logo)) {
                 final ServletContext servletContext = context.getServlet().getServletContext();
-                byte[] logoData = Resources.loadImage(servletContext, appPath,
+                byte[] logoData = resources.loadImage(servletContext, appPath,
                     "images/harvesting/GN3.png", new byte[0]).one();
-                Files.write(logo, logoData);
-            }
+                // Files.write(logo, logoData);
```

Comment:
Why removing this? Initial logo setup is not done now.

Start an empty catalogue and you get
![image](https://user-images.githubusercontent.com/1701393/72142178-66bc1500-3394-11ea-8276-1258ed6816cf.png)

Reply:
See https://github.com/geonetwork/core-geonetwork/pull/4357